### PR TITLE
[HOTFIX] Fix mina demo dockerfile and container

### PR DESCRIPTION
**Pull request by lk86**
Fix demo keys in Dockerfile-rosetta and Dockerfile-coda-demo, update build-rosetta-stages script with opam_branch and mina_branch awareness

- Fixes demo keys in Dockerfile-rosetta
- Fixes Dockerfile-coda-demo
- updates build-rosetta-stages script with opam_branch and mina_branch awareness

### CODE REVIEW

File: scripts/run-demo.sh [hunk 1]
```diff
@@ -5,8 +5,8 @@ set -e
 genesis_time=$(date -d "$(mina advanced compile-time-constants | jq -r '.genesis_state_timestamp')" +%s)
 now_time=$(date +%s)
 
-export CODA_TIME_OFFSET=$(( $now_time - $genesis_time ))
+export CODA_TIME_OFFSET=0
```
Comment (bkase) [inline scripts/run-demo.sh:8]: I thought we need this to start from genesis block even on a build that sets genesis time long in the past

Comment (lk86) [reply scripts/run-demo.sh:8]: these builds set genesis time to like, yesterday, so the complexity here shouldn't be needed, and it was producing a negative value and therefore never producing blocks when figment was testing with it.

**Review by bkase — APPROVED** [review]
If it works, ship it

**Comment by imeckler** [comment]
!approved-for-mainnet

**Pull request merged**